kernel/devices/Rtc.cpp

```diff
@@ -0,0 +1,102 @@
+#include <devices/Rtc.h>
+#include <Cpu.h>
+#include <Log.h>
+
+namespace Kernel::Devices 
+{
+    uint64_t daysPerMonth[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
```

Comment:
I reckon we could mark this as `constexpr`. If you're not familiar with it, it tells the compiler it can evaluate this variable at compile time for optimizations. It would change the variable from `uint64_t daysPerMonth` to `constexpr uint64_t daysPerMonth`, the rest can stay the same.